generators/client/index.js

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Should probably not have blank lines at the start of files - idiomatic style and all.